pkg/config/agentmanagement.go

```diff
@@ -81,6 +88,9 @@ func (r remoteConfigHTTPProvider) FetchRemoteConfig() ([]byte, error) {
 		return nil, fmt.Errorf("error reading remote config: %w", err)
 	}
 
+	// sleep for random time to add jitter to API requests
+	time.Sleep(jitterTime(httpMinJitterTime, httpMaxJitterTime))
```

Comment:
Wouldn't make more sense to add the jitter time to the polling interval?

Reply:
yeah, I think the jitter should be a function of the polling interval, with min and max percentage of the polling interval as parameters (for example, between 0 and 20% of the polling interval).